### PR TITLE
Upgraded to AWS JS SDK v3

### PR DESCRIPTION
**Pull request by ffxsam**
Switched to the new v3 SDK

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -521,7 +519,7 @@ const init = params => {
 
   // Disable ssl if wanted for local development
   if (params.sslEnabled === false) {
-    options.sslEnabled = false
+    options.tls = false
```
Comment (ffxsam) [inline index.js:522]: Not completely sure about this, but `sslEnabled` is definitely gone. It seems like `tls` is the new option now.

See: https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/clients/client-rds-data/interfaces/rdsdataclientconfig.html#tls

**Comment by ffxsam** [comment]
Tests are failing - need to revisit this.

**Comment by brian-learningpool** [comment]
I'm just wondering where things are with this? I've noticed there's also a https://github.com/jeremydaly/data-api-client/tree/aws-sdk-v3-migration branch.

**Comment by ffxsam** [comment]
Since Data API will become legacy at some point, I personally don't see any reason to maintain this library.

Everyone should be migrating over to RDS Proxy ASAP. To my knowledge, the only DB service that supports Data API is Aurora Serverless v1, and since v2 is out, v1's future is uncertain.

**Comment by ataravati** [comment]
@ffxsam Serverless v2 now supports Data API as well.

**Pull request closed without merging**